### ISSUE TITLE
Plugin UI Skewed / Warped in Ableton + Catalina

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/Dplug#512]
Probably graphics regression with the new CGContext method.
Doesn't seem to affect Big Sur though.
Super urgent as the issue is in the wild, reported dozens of minutes after release. 50% of userbase affected.

**Comment by p0nce** [comment]
All hosts, all formats, macOS <= 10.15

**Comment by p0nce** [comment]
Repro script with pgmt + AU + Logic
```
dplug-build -c AU-FREE -a x86_64
rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
sudo cp -R builds/macOS-64b-AU-FREE/*.component /AU/
lldb /Applications/Logic\ Pro\ X.app/Contents/MacOS/Logic\ Pro\ X
```

Could not repro with distort, seems width-dependent

**Comment by p0nce** [comment]
Panagement = no bug
Couture = BUG
Graillon = BUG, 666 pixels width
Distort = no bug, 620 pixels width
Renegate = bug

**Comment by p0nce** [comment]
fc21706fb852a077d136811352e9435d73fa0484 seems like the obvious regression

**Comment by p0nce** [comment]
Can be fixed by reverting fc21706

**Comment by p0nce** [comment]
Doesn't affect Windows. Is only an issue if you used Dplug v10.0.4.
Fixed in v10.0.5, which is for now the only acceptable Dplug v10
